demo/header.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="../assets/css/mason.css">
+  <link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:100,100i,300,300i,400,400i,500,500i,700,700i,900,900i" rel="stylesheet">
+  <script defer src="https://use.fontawesome.com/releases/v5.0.6/js/all.js"></script>
+  <title>Header demo</title>
+  <style>
+  .background-image--demo {
+      height: 300px;
+      background-position: center -20px;
+  }
+  </style>
+</head>
+<body>
+  <header>
+    <h2>Sam Mason Music</h2>
+    <nav class="mason-nav">
+        <ul class="mason-nav__list">
+            <li class="mason-nav__item"><a class="mason-nav__link" href="home.html">Home</a></li>
+            <li class="mason-nav__item"><a class="mason-nav__link" href="music.html">Music</a></li>
+            <li class="mason-nav__item"><a class="mason-nav__link" href="videos.html">Videos</a></li>
+            <li class="mason-nav__item"><a class="mason-nav__link" href="press.html">Press</a></li>
+            <li class="mason-nav__item"><a class="mason-nav__link" href="aboutme.html">About</a></li>
+            <li class="mason-nav__item"><a class="mason-nav__link" href="contact.html">Contact</a></li>
+        </ul>
+        <div class="mason-nav__item"><i class="mason-icon fa-envelope-o"></i></div>
+    </nav>
+</header>
```

Comment:
The indenting is off but perhaps this can be fixed with putting it inside of a `<section>`